firebase-app-distribution/src/test/java/com/google/firebase/app/distribution/NewReleaseFetcherTest.java

```diff
@@ -286,6 +287,31 @@ public void handleNewReleaseFromClient_whenNewAabIsAvailable_returnsRelease() th
         result);
   }
 
+  @Test
+  public void
+      handleNewReleaseFromClient_whenNewReleaseHasDifferentVersionNameThanInstalled_returnsRelease()
+          throws Exception {
+    when(mockFirebaseAppDistributionTesterApiClient.fetchNewRelease(
+            any(), any(), any(), any(), any()))
+        .thenReturn(
+            getTestNewReleaseDifferentVersionName()
+                .setDownloadUrl("http://fake-download-url")
+                .setIasArtifactId(TEST_IAS_ARTIFACT_ID)
+                .setBinaryType(BinaryType.AAB)
+                .build());
```

Comment:
I think you can do something like `getTestInstalledRelease().setDisplayVersion("New Version Name").....`

That way we don't need to create a second method to generate the expected release. 
(This is based on the testing pattern described here: https://testing.googleblog.com/2018/02/testing-on-toilet-cleanly-create-test.html)

Reply:
Great catch! Thanks!